frontend/app/routes/public/renew/$id/adult-child/review-adult-information.tsx

```diff
@@ -374,21 +375,47 @@ export default function RenewAdultChildReviewAdultInformation() {
               </DescriptionListItem>
             </dl>
           </section>
+          {!hasChildren && (
+            <section className="space-y-4">
+              <h2 className="font-lato text-2xl font-bold">{t('renew-adult-child:review-child-information.submit-app-title')}</h2>
+              <p className="mb-4">{t('renew-adult-child:review-child-information.submit-p-proceed')}</p>
+              <p className="mb-4">{t('renew-adult-child:review-child-information.submit-p-false-info')}</p>
+              <p className="mb-4">{t('renew-adult-child:review-child-information.submit-p-repayment')}</p>
```

Comment:
Nitpick: Since these `review-child-information` i18n keys are reserved for the `/review-child-information` route, consider defining new keys for this route.